gatsby-config.js

```diff
@@ -826,101 +414,13 @@ module.exports = {
                             }
                         ]
                     },
-                    {
-                        title: "Troubleshooting guides",
-                        path: "documentation/troubleshooting-guides",
-                        pages: [{
-                                title: "Overview",
-                                path: "documentation/troubleshooting-guides"
-                            },
-                            {
-                                title: "Push messaging",
-                                path: "documentation/troubleshooting-guides/push-messaging"
-                            },
-                            {
-                                title: "Push notifications",
-                                path: "documentation/troubleshooting-guides/push-notifications"
-                            }
-                        ]
-                    },
                     {
                         title: "Frequently asked questions",
                         path: "documentation/faq"
                     },
```

Comment:
let's keep this in aep repo

Reply:
And curate it for AEP specific questions, later maybe.